editor/project_manager.cpp

```diff
@@ -109,9 +109,14 @@ void ProjectManager::_notification(int p_what) {
 			// TRANSLATORS: This refers to the application where users manage their Godot projects.
 			SceneTree::get_singleton()->get_root()->set_title(GODOT_VERSION_NAME + String(" - ") + TTR("Project Manager", "Application"));
 
-			const String line1 = TTR("You don't have any projects yet.");
-			const String line2 = TTR("Get started by creating a new one,\nimporting one that exists, or by downloading a project template from the Asset Library!");
-			empty_list_message->set_text(vformat("[center][b]%s[/b] %s[/center]", line1, line2));
+			empty_list_message->clear();
+			empty_list_message->push_bold();
+			empty_list_message->add_text(TTR("You don't have any projects yet."));
+			empty_list_message->pop();
+			// The bold text above and the regular text below are in the same paragraph, so a space is added.
+			// This space may be redundant when the text is translated (e.g., into Chinese).
+			// It is kept as-is since it's hard to find a better solution.
```

Comment:
I kept rereading, but I am still puzzled by this. "[...] are in the same paragraph, so a space is added"?

Reply:
I mean the space between `[...] yet.` and `Get started by [...]` is necessary. If `Get started by [...]` is on its own line, the space is not needed.

![demo](https://github.com/user-attachments/assets/1b75104f-bf95-4397-ba59-980cde35e110)

Any better wording suggestions?

Reply:
The BBCode was changed here because pseudo-localization would break the BBCode code (e.g. `[b]` could become `[ḅ]`, and in some cases it would dynamically add square brackets, invalidating the syntax).

It is true that pseudo-localization is not intended for end users, but using tag API can be beneficial when debugging the editor.